kubeflow/hack/setup_oauth_client.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# A helper script to create OAuth clients using the caller's GCP project.
+#
+
+if [[ ! -n $ADMIN_EMAIL ]]; then
+  echo "Please provide a value for ADMIN_EMAIL"
+  exit 1
+fi
+
+if [[ ! -n $NAME ]]; then
+  echo "Please provide a value for NAME"
+  exit 1
+fi
+
+BRAND_ID="`gcloud iap oauth-brands list | grep 'name: '`"
+
+if [[ ! -n $BRAND_ID ]]; then
+  echo 'OAuth brand does not exist, creating new one...'
+  BRAND_ID="`gcloud iap oauth-brands create --application_title=kubeflow --support_email=$ADMIN_EMAIL | grep 'name: '`"
+fi
+
+BRAND_ID=$(echo $BRAND_ID | sed -e "s/name: //g")
```

Comment:
Can we remove the back quote symbol '`' from $BRAND_ID assignment, and use `eval` instead of `echo` here?